build_package.py

```diff
@@ -31,6 +31,8 @@ def travis_build_package():
     This method prints on stdout for Travis.
     Return is obj to pass to sys.exit() directly
     """
+    omitted_release_packages = ['azure-keyvault']
```

Comment:
Please make it OMITTED_RELEASE_PACKAGES outside of the method (below DEFAULT_DEST_FOLDER)

Reply:
Done.